### ISSUE TITLE
Doesn't translate selected values in text track settings

### ISSUE THREAD

**Issue by SergeiLovjagin** [videojs/video.js#7639]
## Description

Doesn't translate selected values in text track settings.

## Steps to reproduce

1. Open settings and then text track settings.
2. All titles are translated correctly, but the selected values ​​are not.
3. Open one of the text track settings. All values ​​inside are translated.

### Expected
The selected value of each of the text track settings should be translated.

### Actual
Doesn't translate selected values in text track settings.

**Comment by mister-ben** [comment]
Can you elaborate with an example? [Not all languages in the repo have complete translations](https://github.com/videojs/video.js/blob/main/docs/translations-needed.md), we depend on contributions to keep them up to date.

**Comment by SergeiLovjagin** [comment]
I have added translation to video.js.

` videojs.addLanguage('ru', {
                    White: 'Белый',
                });`
                
In the text track settings, I see the Font Color and the selected color is White (not translated). But when I open the font color selection list, the white color is translated into Russian.

**Comment by mister-ben** [comment]
It's working here: https://codepen.io/mister-ben/pen/XWzMvGQ?editors=1000

It's possibily a timing issue, if translations are added just after the player is initialised. Hard to say without an example.

**Comment by gkatsev** [comment]
Yeah, make sure that translations are added before the player is initialized. Things aren't guaranteed to have the right translations otherwise.
